crates/rpc/rpc/src/eth/api/fees.rs

```diff
@@ -31,6 +32,18 @@ where
         Ok(suggested_tip + U256::from(base_fee))
     }
 
+    /// Returns a suggestion for a gas price for blob transactions.
+    pub(crate) async fn blob_gas_price(&self) -> EthResult<U256> {
+        let header = self.block(BlockNumberOrTag::Latest).await?;
+        Ok(U256::from(
+            header
+                .unwrap_or_default()
+                .excess_blob_gas
+                .ok_or(EthApiError::ExcessBlobGasNotSet)
```

Comment:
I see, since this fee is one-dimensional we don't necessarily need to apply some sampling here and can instead return simply the next fee I think.

we should use 

`Header::next_block_blob_fee` here

Reply:
What do you mean by 1-D and sampling here